<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="push_cam">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>148</y>
      <width>172</width>
      <height>89</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_laser">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>263</y>
      <width>172</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_CCW">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>372</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_CW">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>372</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="opencvFrame">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>580</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>50</y>
      <width>146</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
